src/ui-core/templates/FlashSale/FlashSale.jsx

```diff
@@ -0,0 +1,34 @@
+import React, { useState, useEffect } from "react";
+import "./FlashSale.css";
+import FlexContainer from "../../layouts/FlexContainer/FlexContainer";
+
+import ItemCard from "../../components/molecules/ItemCard/ItemCard";
+
+export default function FlashSale() {
+  const [flashItems, setFlashItems] = useState([]);
+
+  useEffect(() => {
+    async function fetchData() {
+      try {
+        const response = await fetch(
+          "https://fakestoreapi.com/products?limit=5"
+        );
+        const jsonData = await response.json();
+        setFlashItems(jsonData);
+      } catch (error) {
+        console.error("Error fetching data:", error);
+      }
+    }
+    fetchData();
+  }, []);
```

Comment:
@SachinthaSampath Let's do data fetching inside pages—it's better if we pass template data as props and keep it decoupled from the fetch implementation!

- Pages/View is where the final build happens and the rest should be standalone components/templates!
- I think we should move `pages` folder to `ui-core` so that it's more clear